content/editorials/_posts/2018-03-30-is-alphago-zero-overrated.md

```diff
@@ -7,51 +7,50 @@ excerpt: "Why DeepMind's Go playing program is not about to solve all of AI."
 
 <figure>
-      <img src="/content/editorials/images/is-alphago-zero-overrated/history.png" alt="Game History"/>
+      <img src="https://draftin.com:443/images/56473?token=3G6YQX_K8LNCSvkf-fxVAxQMIhG2_8ZOd3uGpjtGKF-myi9JaeEyNf4-cb_r3pynXoHYJr8_n2TPZvaY4OedIjA" alt="Game History"/>
      <figcaption>AlphaGo is the culmination of research into Game AI that stretches all the way back to the birth of AI as a research field. So, it is an inarguably great and historic achievement.<b><a href="http://www.andreykurenkov.com/writing/ai/a-brief-history-of-game-ai/">(source)</a></b></figcaption>
 </figure>
 
 ## Why AlphaGo Zero Is Not That Great
-But moreso, no — AG0 is not really a testament to the usefulness of such techniques for solving the hard problems of AI. You see, Go is only hard within the context of the simplest category of AI problems. That is, it is in the category of problems with every property that makes a learning task easy: it is [deterministic, discrete, static, fully observable, fully-known, single-agent, episodic](https://en.wikibooks.org/wiki/Artificial_Intelligence/AI_Agents_and_their_Environments), cheap and easy to simulate, easy to score... Literally the only challenging aspect of Go is its huge branching factor. Predictions that [AGI (Artificial General Intelligence)](http://theai.wiki/AGI) is imminent based only on AlphaGo's success can be safely dismissed — [the real world is vastly more complex than a simple game like Go](https://medium.com/@karpathy/alphago-in-context-c47718cb95a5). Even fairly similar problems that have most but not all of the properties that make a learning task easy, [such as the strategic video game DotA II](/content/news/openai-dota-ii/), are far beyond our grasp right now. 
+With those positive things having been said, some perspective: AG0 is not really a testament to the usefulness of such techniques for solving the hard problems of AI. You see, Go is only hard within the context of the simplest category of AI problems. That is, it is in the category of problems with every property that makes a learning task easy: it is [deterministic, discrete, static, fully observable, fully-known, single-agent, episodic](https://en.wikibooks.org/wiki/Artificial_Intelligence/AI_Agents_and_their_Environments), cheap and easy to simulate, easy to score... Literally the only challenging aspect of Go is its huge branching factor. Predictions that [AGI (Artificial General Intelligence)](http://theai.wiki/AGI) is imminent based only on AlphaGo's success can be safely dismissed — [the real world is vastly more complex than a simple game like Go](https://medium.com/@karpathy/alphago-in-context-c47718cb95a5). Even fairly similar problems that have most but not all of the properties that make a learning task easy, [such as the strategic video game DotA II](/content/news/openai-dota-ii/), are far beyond our grasp right now. 
```

Comment:
Perhaps "But while in some impressive, some perspective: AG0 is only an iterative improvement in the size of the space we can search in an otherwise 'easy' game." 

Also amusingly this makes me want to write a "Why AI will never solve the truly hard problems in AI" post, about how there's an Overton Window like thing in AI such that what was once "AI" becomes "stuff a computer can do" once computers get decent at it.